CSS/style.css

```diff
@@ -152,8 +174,279 @@ ul {
   .vector {
     margin-left: -10px;
   }
+
+  .icons {
+    width: 30px;
+    size: 25px;
+  }
+}
+
+/* ======================================Work section T====================================================== */
+.work-section {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px;
+  gap: 12px;
+  width: 375px;
+  height: 552px;
+  background: var(--white);
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 1.3rem;
+}
+
+.snapShoot-portfolio {
+  background-image: url(../images/Img/Snapshoot\ Portfolio.png);
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+}
+
+.owner-name {
+  width: 295px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: var(--dark-black);
+  margin: 0.1rem 0 0.1rem 0;
+}
+
+.info {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: stretch;
+  margin: 0.5rem 0 0.5rem 0;
+}
+
+.info h3 {
+  width: 55px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin: 0.1rem 0 0 0;
+}
+
+.info ul li {
+  width: 100px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  list-style-type: square;
+  margin-right: 6px;
+}
+
+.daily-selection {
+  width: 295px;
+  height: 76px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: 0.3rem;
+  padding: 0.1rem;
+}
+
+.list-of-course {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 0.4rem 0 0 -0.3rem;
+  padding: 0.3rem;
+}
+
+.list-of-course a {
+  text-decoration: none;
+  color: #396df2;
+  margin-left: -3px;
+}
+
+.list-of-course a:hover {
+  text-decoration: underline;
+  color: #396df2;
+}
+
+.btn {
+  width: 200px;
+  height: 55px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 0.5rem;
+  padding: 1rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  text-align: center;
+  text-decoration: none;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+}
```

Comment:
- [ ] [OPTIONAL] Please make sure the "See project" button looks similar to the given [FIgma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=t45uZqimwMP5BSMR-0) as I've noticed the width & padding are not In sync with the design.
    - Your design:
       ![firefox_pGgtdol9H1](https://user-images.githubusercontent.com/104968236/229111042-48276b0f-f0dd-434b-b436-e2464283eba9.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/229111196-b8797edd-1d95-4e8f-88c0-9d24d86d2d42.png)
- [ ] [OPTIONAL] Also please make sure you've added the `:hover` and `:active` state to the buttons as shown in the Figma design:
    - Here's the design as a reminder:
       ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/229111513-7f3f7f23-ec7d-46ce-a3cc-29c897da2a40.png)